src/math_private.h

```diff
@@ -261,61 +261,6 @@ irint(double x)
 
 #endif /* __GNUCLIKE_ASM */
 
-/*
- * ieee style elementary functions
- *
- * We rename functions here to improve other sources' diffability
- * against fdlibm.
```

Comment:
unfortunate to drop this. probably worth finding a more general solution to https://github.com/JuliaLang/openlibm/issues/5 here